allennlp/data/vocabulary.py

```diff
@@ -662,6 +681,32 @@ def extend_from_instances(
             min_pretrained_embeddings=min_pretrained_embeddings,
         )
 
+    def _extend_from_transformers_vocabulary(
```

Comment:
Can you move this back inside the token indexer call?  I don't think this is the right fix to #3097.  I'd like to avoid extra parameters that you have to remember to set in the config file; it just makes everything harder to use.  I think there's a different solution that we can come up with, probably just in the timing of when we save the vocab in the trainer code.

Reply:
Done